src/packages.jl

```diff
@@ -215,19 +222,20 @@ For details, please refer to its documentation.
 """
 function install(spec::String, version::String = "";
                                interactive::Bool = true, quiet::Bool = false,
+                               debug::Bool = false,
                                pkgdir::AbstractString = DEFAULT_PKGDIR[])
     # point PackageManager to the given pkg dir
     Globals.PKGMAN_CustomPackageDir = GapObj(pkgdir)
     mkpath(pkgdir)
 
-    if quiet
+    if quiet || debug
       oldlevel = Wrappers.InfoLevel(Globals.InfoPackageManager)
-      Wrappers.SetInfoLevel(Globals.InfoPackageManager, 0)
+      Wrappers.SetInfoLevel(Globals.InfoPackageManager, quiet ? 0 : 3)
```

Comment:
I wonder if it might not be better to separate `debug` and this "verbose" mode: leaving around partial packages could be a stumbling block for non-experts. Yet we may end up asking these very people to use `debug` if they report issues installing packages. Of course we can ask them to `GAP.Packages.remove("blah")` afterwards, but still...

So one idea would be to also add `verbose::Bool = false` as another kwarg. Of course then one can wonder what happens if both `quiet` and `verbose` are set (an error?). Seeing how `quiet` is off by default, perhaps a better alternative would be to add a "verbosity level". So:
- `verbose::Int = 0` is a new kwarg and directly corresponds to the `InfoLevel`
  - alternative names: `verbosity`, `verbosity_level`, `infolevel`, `info_level`, ... ?
  - default value is `0` to match the current default of `quiet=true`
- we either drop `quiet`; or keep it as "deprecated" ("please instead use ...")
  - if set to `true` that sets `verbose` to 0, if set to `false` it sets it to whatever is the default value for `InfoPackageManager` (or whatever value we prefer)
  - implementation:  `function foo(quiet::Bool = true, verbose::Int = (quiet ? 0 : 2))
- thus if both are given, `verbose` wins (we could of course also use `nothing` to detect the case and report an error, but I feel this is overthinking it... on the long run I'd just remove `quiet`)
- we remove the option of "not changing the infolevel (it is redundant if the user has full control over the info level anyway)